database/migrations/1735547965570_create_emails_table.ts

```diff
@@ -0,0 +1,39 @@
+import { BaseSchema } from "@adonisjs/lucid/schema";
+
+export default class ParticipantEmailsSchema extends BaseSchema {
+  protected tableName = "participant_emails";
+
+  async up() {
+    this.schema.createTable(this.tableName, (table) => {
+      table.increments("id");
+      table
+        .integer("participant_id")
+        .unsigned()
+        .references("id")
+        .inTable("participants")
+        .onDelete("CASCADE");
+      table
+        .integer("email_id")
+        .unsigned()
+        .references("id")
+        .inTable("emails")
+        .onDelete("CASCADE");
+      table.timestamp("send_at", { useTz: true });
+      table.text("send_by").nullable();
+      table
+        .enum("status", ["pending", "sent", "failed"], {
+          useNative: true,
+          enumName: "email_status",
+          existingType: false,
+        })
+        .notNullable();
+
+      table.timestamps(true, true); // created_at and updated_at with timezone
```

Comment:
Nie musisz dawać true, true - domyślnie utworzy się created_at i updated_at
```suggestion
      table.timestamps();
```